### DISCUSSION TITLE
Guide to migrate local storage to AWS S3 (minio cluster)

### DISCUSSION THREAD

**Discussion by viceice** [verdaccio/verdaccio#2191] [Q&A]
I like to migrate my verdaccio server from local storage to our minio cluster.

Does anybody have some guidance how to get this best done?

**Comment by juanpicado** [comment]
You might want to ask here, he has quite well experience with minio.
https://github.com/barolab/verdaccio-minio

**Reply by viceice** [reply]
The question is not how to configure verdaccio. It's more how to migrate the private packages.

One way is to update verdaccio config to s3 and  simply republish all packages. but with this i will loose history. 😕

**Reply by juanpicado** [reply]
Unfortunately I haven't heard anyone migrating from a default storage to the cloud and I cannot help since I don't use the cloud. Maybe someone else around here can help you.

**Comment by juanpicado** [comment]
https://twitter.com/_internerd/status/1121899865217339393?s=21
